Automattic/jetpack
# Try: Skip TravisCI if non-testable files are updated

Second attempt at this, previous one was #9677 which was reverted.
TravisCI will attempt to run this in all matrix, but they all should quit fairly quickly instead if no testable files are found.
#### Changes proposed in this Pull Request:

* Updates `.travis.yml` to ignore changesets that just touch non-testable files such as: `.md`, `.txt`, `.yml`, `.*ignore`, `.editorconfig` and `docker/` folder.

#### Testing instructions:



* Check the latest build on TravisCI and make sure they are not run.

If you need to test locally, run the code below after changing files as required:
```
git diff --name-only HEAD~1 | grep -qvE '(\.txt$|\.editorconfig$|\.md$|\.yml$|\..*ignore$)|(^docker)' || {
	echo "Only non-testable files updated, exiting"
	exit
}
echo "testable files updated, running travis"
```
